evm_arithmetization/src/cpu/kernel/asm/transactions/type_3.asm

```diff
@@ -154,8 +154,8 @@ store_origin:
     %eq_const(0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff)
     %jumpi(panic)
 
+global deduct_blob_fee:
```

Comment:
The extra blob fee deduction would fail in the parsing test (since you need to touch the account of the sender, which isn't set). This label allows us to stop the test here (instead of `process_normalized_txn`).

Reply:
You could also have added a sender account in preinitialization of the test FWIW, as here we'd keep this label until we're jumping to `process_normalized_txn` which sounds a bit off.. But well, non-blocking